### DISCUSSION TITLE
Problems with the way we use data and GIS

### DISCUSSION THREAD

**Discussion by ninalong** [albertkun/21S-ASIAAM-191A#46] [Past Thinking Caps]
While seemingly straightforward and objective, mapmaking is a deeply contested space of conquest, conflict, and memory. The names of places, political designations, and border lines reflect political decisions, conscious or unconscious. Does the map in front of you refer to the capital of Vietnam as Saigon, its former and colloquial name, or Ho Chi Minh City, its official post-war designation? This name change is more than simply a result of a toppled regime--it reflects a concerted effort to erase South Vietnam from the collective memory. Among Vietnamese refugees, you will see the exclusive use of Saigon--an effort to remember pre-war roots. Does your map label Taiwan and Hong Kong as independent countries? Where in the South China Sea is the border between China and Taiwan demarcated? Is the sea between Korea and Japan called the Sea of Japan or the East Sea? In the political decisions inherent in mapmaking are both opportunities for maintaining colonial repression and for disrupting the status quo. Mapmakers must think critically about how their craft can uphold democracy, equity, and justice.

**Comment by vcta99** [comment]
That's a great point to bring up -- I hadn't thought to consider how naming conventions are also a reflection of colonialist histories. Your point about the erasure of South Vietnam through the renaming of Saigon especially reminded me of the erasure of Indigenous culture through US naming conventions. This sort of erasure exists even with in UCLA, although there have been recent efforts to increase awareness of the occupied lands we inhabit (the renaming of Janss Steps to Tongva Steps). 

https://dailybruin.com/2020/08/07/usac-unanimously-passes-resolution-in-support-of-renaming-janss-steps-to-tongva-steps

**Comment by alizezila** [comment]
Thanks for sharing! You make a great point about how mappers have an obligation to uphold democracy, equity, and justice. One wouldn't normally think that mapping and justice are connected, but visualizations and representations of countries and populations that have histories of struggle through colonization and imperialism can be misconstrued through mapping. Names in particular are one way that mappers do this.  The question of "Is the sea between Korea and Japan called the Sea of Japan or the East Sea?" is difficult because one must acknowledge the history of Japanese colonization of Korea and how our choices as mappers reflect these long-standing repercussions of struggles for power.
